sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/HttpModelFetcher.cs

```diff
@@ -15,28 +15,28 @@
 namespace Azure.Iot.ModelsRepository.Fetchers
 {
     /// <summary>
-    /// The RemoteModelFetcher is an implementation of IModelFetcher
+    /// The HttpModelFetcher is an implementation of IModelFetcher
     /// for supporting http[s] based model content fetching.
     /// </summary>
-    internal class RemoteModelFetcher : IModelFetcher
+    internal class HttpModelFetcher : IModelFetcher
     {
         private readonly HttpPipeline _pipeline;
         private readonly ClientDiagnostics _clientDiagnostics;
 
-        public RemoteModelFetcher(ClientDiagnostics clientDiagnostics, ModelsRepositoryClientOptions clientOptions)
+        public HttpModelFetcher(ClientDiagnostics clientDiagnostics, ModelsRepositoryClientOptions clientOptions)
         {
             _pipeline = CreatePipeline(clientOptions);
             _clientDiagnostics = clientDiagnostics;
         }
 
         public FetchResult Fetch(
-            string dtmi, Uri repositoryUri, DependencyResolutionOption resolutionOption, CancellationToken cancellationToken = default)
+            string dtmi, Uri repositoryUri, ModelDependencyResolution dependencyResolution, CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope("RemoteModelFetcher.Fetch");
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope("HttpModelFetcher.Fetch");
```

Comment:
you can concatenate name of class and name of method so this comes for free in future refactors (potential refactors)